linux/CMakeLists.txt

```diff
@@ -0,0 +1,139 @@
+# Project-level configuration.
```

Comment:
There are no plans to support linux, macos, web in the near future.
Remove that support. Only keep iOS and Android